kettle/make_json.py

```diff
@@ -312,6 +322,11 @@ def main(db, opts, outfile):
 
     rows_emitted = set()
     for rowid, row in make_rows(db, builds):
+        meterfile = MeterFile()
```

Comment:
this is clever, but does it make more sense to wrap this up in a method that gets the json size and keep the class as an implementation detail?